IdentitySpa/src/app/signup/signup.component.ts

```diff
@@ -0,0 +1,78 @@
+import { Component, ElementRef, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { ApiService } from '../shared/services/api.service';
+import { DelegationService } from '../shared/services/delegation.service';
+import { PasswordMatchValidator } from '../shared/password-match.directive';
+
+@Component({
+  selector: 'app-signup',
+  templateUrl: './signup.component.html',
+  styleUrls: ['./signup.component.scss'],
+})
+export class SignupComponent implements OnInit {
+  signupForm!: FormGroup;
+  signupStatus: {
```

Comment:
There are easier ways to achieve aggregate validations here, but if it works i think we can keep it